test/e2e/pkg/testnet.go

```diff
@@ -154,29 +156,31 @@ func NewTestnetFromManifest(manifest Manifest, file string, ifd InfrastructureDa
 	}
 
 	testnet := &Testnet{
-		Name:                       filepath.Base(dir),
-		File:                       file,
-		Dir:                        dir,
-		IP:                         ipNet,
-		InitialHeight:              1,
-		InitialState:               manifest.InitialState,
-		Validators:                 map[*Node]int64{},
-		ValidatorUpdates:           map[int64]map[*Node]int64{},
-		Nodes:                      []*Node{},
-		Evidence:                   manifest.Evidence,
-		LoadTxSizeBytes:            manifest.LoadTxSizeBytes,
-		LoadTxBatchSize:            manifest.LoadTxBatchSize,
-		LoadTxConnections:          manifest.LoadTxConnections,
-		ABCIProtocol:               manifest.ABCIProtocol,
-		PrepareProposalDelay:       manifest.PrepareProposalDelay,
-		ProcessProposalDelay:       manifest.ProcessProposalDelay,
-		CheckTxDelay:               manifest.CheckTxDelay,
-		VoteExtensionDelay:         manifest.VoteExtensionDelay,
-		FinalizeBlockDelay:         manifest.FinalizeBlockDelay,
-		UpgradeVersion:             manifest.UpgradeVersion,
-		Prometheus:                 manifest.Prometheus,
-		VoteExtensionsEnableHeight: manifest.VoteExtensionsEnableHeight,
-		ExperimentalMaxGossipConnectionsToPersistentPeers:    manifest.ExperimentalMaxGossipConnectionsToPersistentPeers,
+		Name:                             filepath.Base(dir),
+		File:                             file,
+		Dir:                              dir,
+		IP:                               ipNet,
+		InitialHeight:                    1,
+		InitialState:                     manifest.InitialState,
+		Validators:                       map[*Node]int64{},
+		ValidatorUpdates:                 map[int64]map[*Node]int64{},
+		Nodes:                            []*Node{},
+		Evidence:                         manifest.Evidence,
+		LoadTxSizeBytes:                  manifest.LoadTxSizeBytes,
+		LoadTxBatchSize:                  manifest.LoadTxBatchSize,
+		LoadTxConnections:                manifest.LoadTxConnections,
+		ABCIProtocol:                     manifest.ABCIProtocol,
+		PrepareProposalDelay:             manifest.PrepareProposalDelay,
+		ProcessProposalDelay:             manifest.ProcessProposalDelay,
+		CheckTxDelay:                     manifest.CheckTxDelay,
+		VoteExtensionDelay:               manifest.VoteExtensionDelay,
+		FinalizeBlockDelay:               manifest.FinalizeBlockDelay,
+		UpgradeVersion:                   manifest.UpgradeVersion,
+		Prometheus:                       manifest.Prometheus,
+		VoteExtensionsEnableHeight:       manifest.VoteExtensionsEnableHeight,
+		VoteExtensionSize:                manifest.VoteExtensionSize,
```

Comment:
same question

Reply:
You are right. Must have gotten mixed during the cherry-picking.

Reply:
Correct, this came from the original PR, looks like `VoteExtensionSize` is no longer used in QA?